src/pages/index.js

```diff
@@ -0,0 +1,310 @@
+import {
+  enableValidation,
+  settings,
+  resetValidation,
+} from "../scripts/validation.js";
+import "../pages/index.css";
+import headerLogoSrc from "../images/Logo.svg";
+import Api from "../utils/Api.js";
+import { disableButton } from "../scripts/validation.js";
+import { setButtonText } from "../utils/helpers.js";
+
+const headerLogoImg = document.getElementById("header-logo");
+headerLogoImg.src = headerLogoSrc;
+
+// const initialCards = [
+//   {
+//     name: "Val Thorens",
+//     link: "https://practicum-content.s3.us-west-1.amazonaws.com/software-engineer/spots/1-photo-by-moritz-feldmann-from-pexels.jpg",
+//   },
+//   {
+//     name: "Restaurant terrace",
+//     link: "https://practicum-content.s3.us-west-1.amazonaws.com/software-engineer/spots/2-photo-by-ceiline-from-pexels.jpg",
+//   },
+//   {
+//     name: "An outdoor cafe",
+//     link: "https://practicum-content.s3.us-west-1.amazonaws.com/software-engineer/spots/3-photo-by-tubanur-dogan-from-pexels.jpg",
+//   },
+//   {
+//     name: "A very long bridge, over the forest and through the trees",
+//     link: "https://practicum-content.s3.us-west-1.amazonaws.com/software-engineer/spots/4-photo-by-maurice-laschet-from-pexels.jpg",
+//   },
+//   {
+//     name: "Tunnel with morning light",
+//     link: "https://practicum-content.s3.us-west-1.amazonaws.com/software-engineer/spots/5-photo-by-van-anh-nguyen-from-pexels.jpg",
+//   },
+//   {
+//     name: "Mountain house",
+//     link: "https://practicum-content.s3.us-west-1.amazonaws.com/software-engineer/spots/6-photo-by-moritz-feldmann-from-pexels.jpg",
+//   },
+// ];
+
+const api = new Api({
+  baseUrl: "https://around-api.en.tripleten-services.com/v1",
+  headers: {
+    authorization: "c48195d5-e580-4b22-a87c-b06ffed53e6f",
+    "Content-Type": "application/json",
+  },
+});
+
+const profileEditButton = document.querySelector(".profile__edit-button");
+const cardModalButton = document.querySelector(".profile__add-button");
+const editModal = document.querySelector("#edit-modal");
+const profileName = document.querySelector(".profile__name");
+const profileDescription = document.querySelector(".profile__description");
+const profileSubmitButton = document.querySelector(
+  ".modal__submit-button_type_profile"
+);
+
+const avatarModalButton = document.querySelector(".profile__avatar-button");
+const avatarModal = document.querySelector("#edit-avatar-modal");
+const avatarCloseButton = document.querySelector("#avatar-close-button");
+const avatarSubmitButton = document.querySelector("#avatar-submit-button");
+const avatarForm = document.querySelector("#edit-avatar-form");
+const avatarInput = document.querySelector("#avatar-input");
+const profileAvatar = document.querySelector(".profile__avatar");
+
+const editModalCloseButton = editModal.querySelector(".modal__close-button");
+const editFormElement = editModal.querySelector(".modal__form");
```

Comment:
**COULD BE IMPROVED**

We can get any form using their **unique** attribute `name` or `id` from `document.forms`

```js

const profileForm = document.forms["profile-form"];
const cardForm = document.forms["card-form"];

```

You need to add `name` or `id` first in `index.html` to achieve that.

More info here https://developer.mozilla.org/en-US/docs/Web/API/Document/forms